{"name": "test_flaky", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == 4", "trace": "self = <pytest_demo.test_demo_two.TestDemoTwo object at 0x105b89550>\n\n    @pytest.mark.flaky(reruns=5, reruns_delay=3)\n    def test_flaky(self):\n        number = random.randint(1, 5)\n        print(number)\n>       assert number == 4\nE       assert 5 == 4\n\ntest_demo_two.py:53: AssertionError"}, "start": 1599216933745, "stop": 1599216933745, "uuid": "55529b2a-526d-47a7-b496-8953bc968c73", "historyId": "b5ec0b4c98076bdb508a4a95abad03ce", "testCaseId": "10ce1182591748d90873a686451d4318", "fullName": "test_demo_two.TestDemoTwo#test_flaky", "labels": [{"name": "feature", "value": "hotel pytest 分享"}, {"name": "tag", "value": "flaky(reruns=5, reruns_delay=3)"}, {"name": "suite", "value": "test_demo_two"}, {"name": "subSuite", "value": "TestDemoTwo"}, {"name": "host", "value": "wenjian"}, {"name": "thread", "value": "22993-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_demo_two"}]}